### DISCUSSION TITLE
Colored Tabs to keep track of what open books you are currently referencing

### DISCUSSION THREAD

**Discussion by mishaxz** [sumatrapdfreader/sumatrapdf#2724] [Ideas]
It would be great to be able to color a tab (e.g. right click and choose a color). 

Why?

If you have a lot of reference PDFs open, example programming language books, it might be hard to remember which of the 10 books you have open you are currently using. If you could just say assign a color then it is easy to see which books you are currently reading.

**Comment by kjk** [comment]
I've created https://github.com/sumatrapdfreader/sumatrapdf/issues/2725 to track this
